code/modules/jobs/job_types/lawyer.dm

```diff
@@ -16,7 +16,7 @@
 
 	department_flag = CIVILIAN
 	departments = DEPT_BITFLAG_SRV
-	bank_account_department = ACCOUNT_CIV_BITFLAG
+	bank_account_department = ACCOUNT_SRV_BITFLAG
```

Comment:
CIV or SRV? above and below say CIV but these two say SRV

Reply:
My mistake. it is supposed to be civ